modules/installation-aws-about-government-region.adoc

```diff
@@ -19,6 +18,15 @@ The following AWS GovCloud partitions are supported:
 * `us-gov-west-1`
 * `us-gov-east-1`
 
-The AWS GovCloud region and custom AMI must be manually configured in the
+The following AWS Secret Region partition is supported:
+
+* `us-iso-east-1`
+
+The AWS government or secret region, and accompanying custom AMI, must be manually configured in the
 `install-config.yaml` file since {op-system} AMIs are not provided by Red Hat
 for those regions.
+
+[IMPORTANT]
+====
+If you are deploying to the C2S Secret Region, you must also define a custom CA certificate in the `additionalTrustBundle` field of the `install-config.yaml` file, since the AWS API requires a custom CA trust bundle. To allow the installation program to access the AWS API, the CA certificates must also be defined on the machine running the installation program. This can be done by adding the CA bundle to the trust store on the machine, using the `AWS_CA_BUNDLE` environment variable, or defining the CA bundle in the link:https://docs.aws.amazon.com/credref/latest/refdocs/setting-global-ca_bundle.html[`ca_bundle`] field of the AWS config file.
```

Comment:
```suggestion
If you are deploying to the C2S Secret Region, you must also define a custom CA certificate in the `additionalTrustBundle` field of the `install-config.yaml` file because the AWS API requires a custom CA trust bundle. To allow the installation program to access the AWS API, the CA certificates must also be defined on the machine that runs the installation program. You must add the CA bundle to the trust store on the machine by defining the `AWS_CA_BUNDLE` environment variable or defining the CA bundle in the link:https://docs.aws.amazon.com/credref/latest/refdocs/setting-global-ca_bundle.html[`ca_bundle`] field of the AWS config file.
```